### ISSUE TITLE
Static component import not working

### ISSUE THREAD

**Issue by theonelucas** [faceyspacey/react-universal-component#191]
According to this line of code, the component can either be a string or a component:
```
typeof page === 'string' ? import(`./${page}`) : () => page()
```

But when we try to import the component:

```
import Page1 from '../components/Page1'

const components = {
  HOME: 'Home',
  PAGE1: Page1,
  [NOT_FOUND]: 'NotFound'
}
```

An error is being returned:
```
Error: load is not a function
```

How the component should be imported?

**Comment by tstirrat15** [comment]
Where is your `import` statement?

**Comment by theonelucas** [comment]
> Where is your `import` statement?

`import Page1 from '../components/Page1'
`

**Comment by tstirrat15** [comment]
Right, that's a synchronous import that isn't covered by this component. The `import()` function and the `import x from 'y'` statement are two different things. To use this component, you need to be using the function:

```javascript
const ListingPage = universal(import('./listing/components/Root'), { onError: exceptionHandler });
```
That sort of thing.

**Comment by theonelucas** [comment]
So, basically I cannot pass a component already imported to Universal?

**Comment by theonelucas** [comment]
That makes sense, but what I'm aiming here is to do an incremental code splitting. Some routes, which I would explicitly pass an previously imported component, would just use it, but the other routes with a string as the component would fetch it at runtime. That way I can really leverage the power of code splitting, bundling some pages that will always be used and fetching the others as needed.
